config.js

```diff
@@ -179,11 +179,6 @@ var conf = convict({
       format: String,
       default: 'https://app.adjust.com/2uo1qc?campaign=fxa-conf-email&adgroup=ios&creative=button&fallback=https%3A%2F%2Fitunes.apple.com%2Fapp%2Fapple-store%2Fid989804926%3Fpt%3D373246%26ct%3Dadjust_tracker%26mt%3D8'
     },
-    signInUrl: {
```

Comment:
I believe removing this here will also require a corresponding [change to the auth-server](https://github.com/mozilla/fxa-auth-server/blob/master/config/index.js#L227) and [here too](https://github.com/mozilla/fxa-auth-server/blob/master/config/index.js#L495), and possibly puppet-config.